channels/drive/client/drive_main.c

```diff
@@ -274,10 +274,13 @@ static UINT drive_process_irp_read(DRIVE_DEVICE* drive, IRP* irp)
 	if (!drive || !irp || !irp->output || !irp->Complete)
 		return ERROR_INVALID_PARAMETER;
 
+	if (Stream_GetRemainingLength(irp->input) < 6)
```

Comment:
I think that should be < 12

Reply:
yes of course !

Reply:
@hardening Sure you don't want to change that too?